### DISCUSSION TITLE
TrackInfo button

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1137] [Team Posts]
I hadn't noticed this was a button for a long time:
![image](https://user-images.githubusercontent.com/10009541/70502435-03369180-1b21-11ea-934d-2d30bf7d6f21.png)
It does not look good (slightly cut-off on both sides, wrongly written without space, was possibly supposed to have the same width as the mixer strip minimum? - but it does not work with translations anyway...). And is it really track info? Or rather a mixer strip for the current track? 
Perhaps an icon button with tooltip (but in the same fixed width as it is now) would look better?

**Comment by terminator356** [comment]
Ye olde ancient prehistory there. From the cro-magnon era I believe.
Sit by the fire and I'll tell the tale.
Once upon a (another, different) time in MusE there was only one kind of side-panel.
MusE was mostly midi, I believe it had no audio, and this side panel was called...
"Midi Track Info".
This panel was all text boxes, no real graphics.
As audio was added, there still did not exist for a long time a 'panel' for it.
We had a mixer strip but not a 'panel' yet.
Then I replaced the Midi Track Info panel with the midi mixer strip.
Eventually I added the audio mixer strip. Relatively recently speaking.
Never got around to that silly button. Be my guest, give it a new 'do' of some kind. Icon, me like.
Believe it or not for a few years there were **two** panels and two buttons !
The old panel module "mtrackinfo.cpp/h" still exists.
By now it is rusting away like a ship at the bottom of the sea.
I suppose we could bury it, give a proper sendoff. "We salute thee, Midi Track Info !"

**Comment by terminator356** [comment]
That second 'alt' button I mentioned was still in the code.
I've now removed it and all refs to it.
